packages/tpetra/core/src/Tpetra_Details_Behavior.cpp

```diff
@@ -84,6 +163,47 @@ namespace { // (anonymous)
     return value;
   }
 
+  void
+  idempotentlyGetEnvironmentVariableAsMap (std::once_flag& once_flag,
```

Comment:
Since `valsMap won't get filled in if this function has been called before, it might be better to call this `getEnvironmentVariableAsMapOnce`.  (It's not technically "idempotent" if calling it the second time gives a different result than calling it the first time.)  This is just a quibble over naming, not a correctness issue (but see comments below).

Reply:
okay